Example/PaymentSheet Example/PaymentSheet Example/PaymentSheetTestPlaygroundSettings.swift

```diff
@@ -0,0 +1,270 @@
+        static var enumName: String { "Mode" }
+
+        case payment
+        case paymentWithSetup = "payment_with_setup"
+        case setup
+
+        var displayName: String {
+            switch self {
+            case .payment:
+                return "Payment"
+            case .paymentWithSetup:
+                return "Pay+SFU"
+            case .setup:
+                return "Setup"
+            }
+        }
+    }
+
+    enum IntegrationType: String, PickerEnum {
+        static var enumName: String { "Type" }
+
+        // Normal: Normal client side confirmation non-deferred flow
+        case normal
+        /// Def CSC: Deferred client side confirmation
+        case deferred_csc
+        /// Def SSC: Deferred server side confirmation
+        case deferred_ssc
+        /// Def MC: Deferred server side confirmation with manual confirmation
+        case deferred_mc
+        /// Def MP: Deferred multiprocessor flow
+        case deferred_mp
+
+        var displayName: String {
+            switch self {
+            case .normal:
+                return "Client-side confirmation"
+            case .deferred_csc:
+                return "Deferred client side confirmation"
+            case .deferred_ssc:
+                return "Deferred server side confirmation"
+            case .deferred_mc:
+                return "Deferred server side confirmation with manual confirmation"
+            case .deferred_mp:
+                return "Deferred multiprocessor flow"
+            }
+        }
+    }
+
+    enum CustomerMode: String, PickerEnum {
+        static var enumName: String { "Customer" }
+
+        case guest
+        case new
+        case returning
+    }
+
+    enum Currency: String, PickerEnum {
+        static var enumName: String { "Currency" }
+
+        case usd
+        case eur
+        case aud
+        case gbp
+        case inr
+    }
+
+    enum MerchantCountry: String, PickerEnum {
+        static var enumName: String { "MerchantCountry" }
+
+        case US
+        case GB
+        case AU
+        case FR
+        case IN
+    }
+
+    enum APMSEnabled: String, PickerEnum {
+        static var enumName: String { "Automatic PMs" }
+
+        case on
+        case off
+    }
+
+    enum ShippingInfo: String, PickerEnum {
+        static var enumName: String { "Shipping info" }
+
+        case on
+        case onWithDefaults = "on w/ defaults"
+        case off
+    }
+
+    enum ApplePayEnabled: String, PickerEnum {
+        static var enumName: String { "Apple Pay" }
+
+        case on
+        case off
+        case onWDetails = "on w/details"
+    }
+
+    enum ApplePayButtonType: String, PickerEnum {
+        static var enumName: String { "Pay button" }
+
+        case plain
+        case buy
+        case setup
+        case checkout
+    }
+
+    enum AllowsDelayedPMs: String, PickerEnum {
+        static var enumName: String { "allowsDelayedPMs" }
+
+        case `true`
+        case `false`
```

Comment:
feel free to disregard but, if we want to avoid using the escape ticks, we could use enabled/disabled, or on/off

Reply:
Sounds good, I was trying to be consistent with the original implementation but on/off does seem better